includes/data-stores/class-wc-data-store-posts.php

```diff
@@ -0,0 +1,27 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Shared logic for posts data stores.
+ *
+ * @version  2.7.0
+ * @category Class
+ * @author   WooThemes
+ */
+class WC_Data_Store_Posts  {
```

Comment:
I think `_CPT` fits more then `Custom Content Type` since the system in WordPress is specifically called custom **post** type, so I think we should go with that.